packages/contracts/src/dollar/core/UbiquityCreditToken.sol

```diff
@@ -31,18 +41,22 @@ contract UbiquityCreditToken is ERC20Ubiquity {
         _;
     }
 
-    /// @notice raise capital in form of Ubiquity Credit Token (only redeemable when Ubiquity Dollar > 1$)
-    /// @param amount the amount to be minted
-    /// @dev you should be minter to call that function
+    /**
+     * @notice Raises capital in the form of Ubiquity Credit Token (only redeemable when Ubiquity Dollar > 1$)
```

Comment:
```suggestion
     * @notice Raises capital in the form of Ubiquity Credit Token 
```

The redemption logic is subject to change in the future, but the original (perhaps naive) idea was that if we can afford to print more dollars, then we should set aside value to allow 100% of credits to be redeemed.